skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -22,19 +22,15 @@
 from future.utils import with_metaclass
 import json
 
-from future import standard_library
-standard_library.install_aliases()
-from builtins import str
-from builtins import range
-
 # SKA specific imports
 file_path = os.path.dirname(os.path.abspath(__file__))
 auxiliary_path = os.path.abspath(os.path.join(file_path, os.pardir)) + "/auxiliary"
 sys.path.insert(0, auxiliary_path)
 from skabase import release
-from utils import (get_dp_command, exception_manager,
-                           tango_type_conversion, coerce_value,
-                           get_groups_from_json, get_tango_device_type_id)
+from utils import (get_dp_command,
+                   coerce_value,
+                   get_groups_from_json,
+                   get_tango_device_type_id)
 
 from faults import GroupDefinitionsError
 import logging
```

Comment:
did you try the tmc without logging?

Reply:
We are working on this but as part of separate item in TMC prototype. Since current PR is only to address Pylint issue, I am marking this conversation as Resolved here.